acceptance/current/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'kitchen destroy' do
+  cwd File.join(File.dirname(__FILE__), "../../../../../../../..")
```

Comment:
Definitely agree with the comment @schisamo. We will address that in chef-acceptance first and make the necessary changes in mixlib-install once it is done. I have captured that feedback in GS-16.